src/groups/mqb/mqbblp/mqbblp_cluster.cpp

```diff
@@ -2929,6 +2933,24 @@ void Cluster::processControlMessage(
             << description() << ": unexpected clusterMessage:" << message
             << MWCTSK_ALARMLOG_END;
     } break;  // BREAK
+    case MsgChoice::SELECTION_ID_ADMIN_COMMAND: {
+        const bmqp_ctrlmsg::AdminCommand& adminCommand =
+            message.choice().adminCommand();
+        const bsl::string& cmd      = adminCommand.command();
+        const bool         rerouted = adminCommand.rerouted();
```

Comment:
Is there a case when `rerouted` is `false`?

Reply:
I was wondering this too. I don't think there is, which would mean the `rerouted` field is essentially useless and can be removed. The only reason for keeping this might be to "future-proof" it. I'm not sure if there is any possible use case for sending admin commands between nodes other than in the context of "rerouting". It seems not.